### ISSUE TITLE
'CTranModel' object has no attribute 'module'. Did you mean: 'modules'?

### ISSUE THREAD

**Issue by ans92** [QData/C-Tran#20]
Hi,
I want to run the model by freezing the backbone I am getting the following error in main.py line 56:
`'CTranModel' object has no attribute 'module'. Did you mean: 'modules'?`

When I change module to modules then I get following error on same line 56 in main.py:
`AttributeError: 'function' object has no attribute 'backbone'
`

Could you please help me regarding this?

**Comment by jacklanchantin** [comment]
Can you share the command line you are using? Are you using GPUs?

**Comment by ans92** [comment]
Thank you for your reply
Yes I am using GPU. I am using following command line:
```
python3 "/home/ans/CZSL/C-Tran-main/main.py"  --batch_size 7  --lr 0.00001 --optim 'adam' --layers 3  --dataset 'mit_states' --grad_ac_step 2 --dataroot data/ --epochs 100 --freeze_backbone
```

**Comment by jacklanchantin** [comment]
And were you able to run the commands in the README?

**Comment by ans92** [comment]
Yes I were able to download and run the model on voc20 dataset. Even I am able to run the model on new dataset when I also update the image extractor. However, if I keep the image extractor fixed through --freeze_backbone parameter then this error occurs.

**Comment by jacklanchantin** [comment]
Can you try running without a GPU?

**Comment by ans92** [comment]
Yes I have run the model with cpu but error still exists.

**Comment by jacklanchantin** [comment]
It could be. I can't replicate the error

**Comment by ans92** [comment]
Is it possible for you to upgrade your torch and torchvision library to the latest version as I have mentioned above?
